packages/cwp-template-basic/README.md

```diff
@@ -0,0 +1,7 @@
+# cwp-template
```

Comment:
I'm aware this was just a c/p, but while we're at it, let's update this README with some meaningful info. You can do that for the cms/full templates as well.

Reply:
should i add, the service/apis it have and how to make use of it, like similar to the things mentioned in PR

Reply:
Yeah why not.

Explain what this template consists of and why it exists - for the users that don't need any of the official Webiny apps (PB,FB,CMS...), and just want to start building their own custom app.